flocker/node/agents/test/test_blockdevice.py

```diff
@@ -844,6 +844,15 @@ class IBlockDeviceAPITestsMixin(object):
     """
     Tests to perform on ``IBlockDeviceAPI`` providers.
     """
+    def tearDown(self):
+        """
+        Destroy volumes created for the cluster used by the test.
+        """
+        volumes = self.api.list_volumes()
+
+        for volume in volumes:
+            self.api.destroy_volume(volume.blockdevice_id)
```

Comment:
If this remains, we'll need to make some changes to the existing loopback block device tests...I think...but let me quickly check...